File: C++/sandbox.cpp
 #include <iostream>

int one() {
    return 1;
}

int two() {
    return 2;
}

int three() {
    return 3;
}

int four() {
    return 4;
}

int five() {
    return 5;
}

int six() {
    return 6;
}

int seven() {
    return 7;
}

int eight() {
    return 8;
}

int nine() {
    return 9;
}

int ten() {
    return 10;
}

int addNumber(int num, int numToAdd) {
    return num + numToAdd;
}

int main() {
    std::cout << "Hello, World!";
    int result = addNumber(2, 2);
    std::cout << result;
    return 0;        
}

File: C#/sandbox.cs
using System;

class Program {
    
    static int One() {
        return 1;
    }

    static int Two() {
        return 2;
    }

    static int Three() {
        return 3;
    }

    static int Four() {
        return 4;
    }

    static int Five() {
        return 5;
    }

    static int Six() {
        return 6;
    }

    static int Seven() {
        return 7;
    }

    static int Eight() {
        return 8;
    }

    static int Nine() {
        return 9;
    }

    static int Ten() {
        return 10;
    }    

    static int AddNumber(int num, int numToAdd) {
        return num + numToAdd;
    }
    
    static void Main(string[] args) {
        Console.WriteLine("Hello, World!");
        int result = AddNumber(2, 2);
        Console.WriteLine(result);
    }

}
